inventory/byo/hosts.example

```diff
@@ -615,6 +615,71 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 #openshift_logging_image_prefix=registry.access.redhat.com/openshift3/
 #openshift_logging_image_version=3.7.0
 
+# Prometheus deployment
+#
+# Currently prometheus deployment is disabled by default, enable it by setting this
+#openshift_hosted_prometheus_deploy=true
+#
+# Prometheus storage config
+# Option A - NFS Host Group
+# An NFS volume will be created with path "nfs_directory/volume_name"
+# on the host within the [nfs] host group.  For example, the volume
+# path using these options would be "/exports/prometheus"
+#openshift_prometheus_storage_kind=nfs
+#openshift_prometheus_storage_access_modes=['ReadWriteOnce']
+#openshift_prometheus_storage_nfs_directory=/exports
+#openshift_prometheus_storage_nfs_options='*(rw,root_squash)'
+#openshift_prometheus_storage_volume_name=prometheus
```

Comment:
README requires variable documentation updates to describe all the new config parameters 

If some of these are not intended to be overridden by the users (e.g., *constants*) then they should be moved into the `vars/` directory and may not require additional documentation.

Reply:
@tbielawa I started changing the readme but then I realized these variables are not in this role scope, and shouldn't be added in the role README.
If any, this README should update:
https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_storage_nfs
but, it currently doesn't have all the logging, metrics, etc vars in it.

Was that the one you meant?